tests/components/test_websocket_api.py

```diff
@@ -297,7 +297,7 @@ def test_get_panels(hass, websocket_client):
     hass.data[frontend.DATA_JS_VERSION] = 'es5'
     yield from websocket_client.send_json({
         'id': 5,
-        'type': wapi.TYPE_GET_PANELS,
+        'type': 'get_panels',
     })
```

Comment:
Ideally this test would now be in `test_frontend.py`, right?

Reply:
Yes, I'll move it.